src/commands/HBonds.cpp

```diff
@@ -191,7 +191,7 @@ int HBonds::run(int argc, const char* argv[]) {
         }
 
         outfile << "# Frame: " << step << std::endl;        
-        outfile << "# Donor   Acceptor   Hydrogen  : Distance D-A    Angle D-A-H" << std::endl;
+        outfile << "# Donor (name index)   Acceptor (name index)   Hydrogen (name index)  : Distance D-A    Angle D-A-H" << std::endl;
```

Comment:
no, imagine this configuration:
                    D
   A-----H  
We want A, H and D to be the "most co-linear" possible. If the angle D-A-H is zero, then they are aligned, if not <e want the angle to be less than 30°.
By taking D-H-A, we would want it to be equal to 180° +/- 30°.